.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Description
+
+    Describe your changes
+
+## Reference
+
+    i.e Jira or Github issue URL
+
+## Type Of Change
+
+- [ ] Bug Fix
+- [ ] New Feature 
```

Comment:
This should not be needed. We have the jira types for this. If we want to include this, it should be in sync with the jira types.